hack/build-images.sh

```diff
@@ -18,6 +18,9 @@ source "${OS_ROOT}/hack/util.sh"
 source "${OS_ROOT}/contrib/node/install-sdn.sh"
 os::log::install_errexit
 
+OS_LOGGING_BRANCH=${OS_LOGGING_BRANCH:-master}
+OS_METRICS_BRANCH=${OS_METRICS_BRANCH:-master}
```

Comment:
This value needs to be coming from a checked in file in our repo (or a git subrepo), something that identifies an exact commit.  We want to go for reproducibility.